vtr_flow/tasks/regression_tests/vtr_reg_strong/strong_sub_tiles/config/config.txt

```diff
@@ -0,0 +1,28 @@
+##############################################
```

Comment:
Thanks for adding the test case. However I don't see the associated golden_results.txt file in the PR. It should be included (you can generate it with parse_vtr_task.pl -create_golden after running the task).

Reply:
Done